boms/cloud-oss-bom/pom.xml

```diff
@@ -49,19 +49,19 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <guava.version>31.1-jre</guava.version>
     <gson.version>2.9.0</gson.version>
-    <google.cloud.bom.version>0.172.0</google.cloud.bom.version>
-    <google.cloud.core.version>2.5.11</google.cloud.core.version>
-    <io.grpc.version>1.45.0</io.grpc.version>
-    <http.version>1.41.5</http.version>
+    <google.cloud.bom.version>0.173.0</google.cloud.bom.version>
+    <google.cloud.core.version>2.6.0</google.cloud.core.version>
+    <io.grpc.version>1.45.1</io.grpc.version>
+    <http.version>1.41.7</http.version>
```

Comment:
The shared dependencies BOM 2.10.0

https://github.com/googleapis/java-shared-dependencies/blob/v2.10.0/first-party-dependencies/pom.xml#L58